spec/system/placements/sign_in_as_a_placements_user_spec.rb

```diff
@@ -74,7 +74,7 @@
   end
 
   context "when normal user tries to access support user page" do
-    scenario "I sign in as user mary trying to access support user page" do
+    scenario "I sign in as user mary trying to access support user page", :js do
```

Comment:
Does this need to be here?

```suggestion
    scenario "I sign in as user mary trying to access support user page" do
```